src/green/Green.jl

```diff
@@ -264,6 +273,44 @@ function toDLR(green::Green2DLR, targetGrid = green.dlrGrid.ω)
 end
 
 
+"""
+    function getValue(green::Green2DLR, timevalue, spacevalue, colorvalue, spacemethod=:linear)
+
+Find value of Green's function at given color, τ/ωn and k/x by interpolation.
+Interpolation in τ/ωn use DLR method
+#Argument
+- 'green': Green's function
+- 'timevalue': Target τ/ωn point 
+- 'spacevalue': Target k/x point
+- 'colorvalue': Target color
+- spacemethod: Method of interpolation in k/x 
+"""
+function getValue(green::Green2DLR, timevalue, spacevalue, colorvalue=nothing, spacemethod=:linear)
+    @assert  green.timeType == :n ||green.timeType == :τ
+
+    if green.hasError
+        #error = green.dynamicError
+        error = nothing
+    else
+        error = nothing
+    end
+    if(isnothing(colorvalue))
+        cindex = 1
+    else
+        cindex = green.color.getIndex(colorvalue)
+    end
+    dynamic_c = green.dynamic[cindex, cindex, :, :]
+
+    dynamic_x = green.spaceGrid.getValue(dynamic_c, spacevalue, spacemethod, axis = 1)
```

Comment:
use interp1D(data, xgrid::T, x; axis=1) from CompositeGrids.Interp

Reply:
getValue --> get
(green::Green2DLR, timevalue, spacevalue, colorvalue=nothing, spacemethod=:linear) -->  (green::Green2DLR, time, space; color=nothing, timeMethod=:linear, spacemethod=:linear)

Reply:
dynamic_x = CompositeGrids.Interp.interp1D(dynamic_c, green.spaceGrid, spacevalue; axis=1, interpstyle=:default)
interpstyle=:linear for linear interp